packages/datepickers/src/styled/StyledHeaderPaddle.ts

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { retrieveComponentStyles, getColor, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+import { retrieveSpacing } from './utils';
+
+const COMPONENT_ID = 'datepickers.header_paddle';
+
+const StyledHeaderPaddle = styled.div.attrs({
+  'data-garden-id': COMPONENT_ID
+})<{
+  isSmall: boolean;
+  isHidden?: boolean;
+}>`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transform: ${props => props.theme.rtl && 'rotate(180deg)'};
+  border-radius: 50%;
+  cursor: pointer;
+  width: ${retrieveSpacing};
+  height: ${retrieveSpacing};
+
+  color: ${props => getColor('neutralHue', 600, props.theme)};
+
+  ${props => props.isHidden && `visibility: hidden;`};
```

Comment:
Pull `visibility` out so that properties can be read together as `property: value`. In this case `visibility: ${props => props.isHidden && 'hidden'}`. Don't remove it from the ordering of the base/core properties above.

Reply:
I think in a previous version of `styled-components` this would have actually rendered `visibility: false`. It looks like this has been cleaned up. Updated.